app/templates/server/server.js

```diff
@@ -14,6 +14,13 @@ const serviceManager = require('./services/service-manager');
 require('./services/index')(app);
 require('./routers/index')(app);
 
+// Uncomment following to enable zipkin tracing, tailor to fit your network configuration:
+// var appzip = require('appmetrics-zipkin')({
```

Comment:
Doesn't the `require('appmetrics-zipkin')` statement need to go before the other `require(appmetrics-x)` statements? i.e. before line 1?

Reply:
We might want to link in a comment to some docs or tutorial here for how to setup zipkins. In order to use it, user needs to get and start the zipkin server, then add http(s) requests to see tracing, if I understand zipkins correctly.

Reply:
@joyychang I've confirmed you need to move the block that starts at line 17 up before the requires on line 1.